### PR TITLE
Add Huawei AppGallery Installer ID enum

### PR DESCRIPTION
**Pull request by lahdekorpi**
Implements #77

### CODE REVIEW

**Comment by jahirfiquitiva** [comment]
1. CI build is failing. [Check details here](https://travis-ci.org/javiersantos/PiracyChecker/builds/636304260)
2. Please rename it to `HUAWEI_APP_GALLERY`

File: app/src/androidTest/java/com/github/javiersantos/piracychecker/PiracyCheckerTest.java [hunk 1]
```diff
@@ -108,6 +108,7 @@ public void run() {
                     .enableInstallerId(InstallerID.GOOGLE_PLAY)
                     .enableInstallerId(InstallerID.AMAZON_APP_STORE)
                     .enableInstallerId(InstallerID.GALAXY_APPS)
+                    .enableInstallerId(InstallerID.HUAWEI_APPGALLERY)
```
Comment (jahirfiquitiva) [inline app/src/androidTest/java/com/github/javiersantos/piracychecker/PiracyCheckerTest.java]:
```suggestion
                    .enableInstallerId(InstallerID.HUAWEI_APP_GALLERY)
```

File: library/src/main/java/com/github/javiersantos/piracychecker/enums/InstallerID.kt [hunk 2]
```diff
@@ -7,6 +7,7 @@ enum class InstallerID(private val text: String) {
     GOOGLE_PLAY("com.android.vending|com.google.android.feedback"),
     AMAZON_APP_STORE("com.amazon.venezia"),
     GALAXY_APPS("com.sec.android.app.samsungapps");
+    HUAWEI_APPGALLERY("com.huawei.appmarket");
```
Comment (jahirfiquitiva) [inline library/src/main/java/com/github/javiersantos/piracychecker/enums/InstallerID.kt]:
```suggestion
    HUAWEI_APP_GALLERY("com.huawei.appmarket");
```

File: library/src/main/java/com/github/javiersantos/piracychecker/enums/InstallerID.kt [hunk 3]
```diff
@@ -7,6 +7,7 @@ enum class InstallerID(private val text: String) {
     GOOGLE_PLAY("com.android.vending|com.google.android.feedback"),
     AMAZON_APP_STORE("com.amazon.venezia"),
     GALAXY_APPS("com.sec.android.app.samsungapps");
```
Comment (jahirfiquitiva) [inline library/src/main/java/com/github/javiersantos/piracychecker/enums/InstallerID.kt]:
```suggestion
    GALAXY_APPS("com.sec.android.app.samsungapps"),
```

**Comment by lahdekorpi** [comment]
My bad on the syntax. Sorry, I forgot about this PR earlier. Thanks a lot for fixing it!

**Comment by jahirfiquitiva** [comment]
[Builds are failing](https://travis-ci.org/github/javiersantos/PiracyChecker/builds/667252895) ... mind checking tests, please?

**Pull request merged**